### PR TITLE
Review descriptions for initial duplication.

### PR DESCRIPTION
**Pull request by Insti**
Pull Request #775 prepended a problem's blurb to its description. This resulted in a few descriptions having obvious duplication in the first few lines.

This PR reviews all descriptions and:
* Removes gratuitous duplication.
* Ensures the first line does not exceed 80 characters.
* Fixes a few spelling/grammar errors.
* Re-arranges a few descriptions for clarity.
* Removes a few occurrences of 'write a program'

Thanks to @ErikSchierboom for [pointing many of these out](https://github.com/exercism/x-common/pull/775#pullrequestreview-39794067).

Dependency: 
- [x] Completed - https://github.com/exercism/trackler/pull/49. Merging before this is completed may result in the appearance of first line duplication when the readmes for these problems are generated.

Refs:
[PR #775](https://github.com/exercism/x-common/pull/775)
[Write a program](https://github.com/exercism/x-common/issues/321)

### CODE REVIEW

File: exercises/pangram/description.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 Determine if a sentence is a pangram. A pangram (Greek: παν γράμμα, pan gramma,
 "every letter") is a sentence using every letter of the alphabet at least once.
-The best known English pangram is "The quick brown fox jumps over the lazy dog."
+The best known English pangram is "The quick brown fox jumps over the lazy
+dog."
```
Comment (Insti) [inline exercises/pangram/description.md]: Try and avoid line wrapping inside quotes.

File: exercises/perfect-numbers/description.md [hunk 2]
```diff
@@ -1,4 +1,5 @@
-Determine if a number is perfect, abundant, or deficient based on Nicomachus' (60 - 120 CE) classification scheme for natural numbers.
+Determine if a number is perfect, abundant, or deficient based on Nicomachus'
+(60 - 120 CE) classification scheme for natural numbers.
```
Comment (Insti) [inline exercises/perfect-numbers/description.md]: Try and keep Nicomachus' dates with his name.

File: exercises/diamond/description.md [hunk 3]
```diff
@@ -1,5 +1,3 @@
-Given a letter, print a diamond starting with 'A' with the supplied letter at the widest point.
-
 ## Diamond kata
```
Comment (ErikSchierboom) [inline exercises/diamond/description.md]: Does the heading make sense? It is rarely used elsewhere.

File: exercises/book-store/description.md [hunk 4]
```diff
@@ -1,5 +1,3 @@
-To try and encourage more sales of different books from a popular 5 book series, a bookshop has decided to offer discounts of multiple-book purchases.
-
```
Comment (rpottsoh) [inline exercises/book-store/description.md]: @ErikSchierboom I have been following the discussion here and I agree with you on this point but I would like to add that the hyphen should be removed if `multiple` is left in.  I like `multiple book` better than `multi-book`.  I never really noticed it before till the topic of consolidation came up.

File: exercises/book-store/description.md [hunk 5]
```diff
@@ -1,8 +1,5 @@
...
-To try and encourage more sales of the 5 different books
-they sell of a popular series, a bookshop has decided to
-offer discounts of multi-book purchases. 
+To try and encourage more sales of different books from a popular 5 book
+series, a bookshop has decided to offer discounts of multiple book purchases.
```
Comment (Insti) [inline exercises/book-store/description.md]: Should this be "**on** multiple book purchases"?

File: exercises/book-store/description.md (hunk 4 above)
Comment (Insti) [inline exercises/book-store/description.md]: Replaced and de-hyphened as suggested.

**Comment by rpottsoh** [comment]
I think ‘on’ rolls off the tongue better.

**Comment by Insti** [comment]
Barring any huge mistakes we've not found, this is ready to be squashed and merged now.
Further tweaking of descriptions can happen in separate specific PRs.

File: exercises/octal/description.md [hunk 6]
```diff
@@ -1,4 +1,6 @@
-Convert a octal number, represented as a string (e.g. '1735263'), to its decimal equivalent using first principles (i.e. no, you may not use built-in or external libraries to accomplish the conversion).
+Convert a octal number, represented as a string (e.g. '1735263'), to its
```
Comment (stkent) [inline exercises/octal/description.md]: > Convert a octal number

should probably read

> Convert an octal number

(This exercise is deprecated but likely still served by some tracks, so I figure the change is worth making?)

File: exercises/sum-of-multiples/description.md [hunk 7]
```diff
@@ -1,4 +1,5 @@
-Given a number, find the sum of all the multiples of particular numbers up to but not including that number.
+Given a number, find the sum of all the multiples of particular numbers up to
```
Comment (stkent) [inline exercises/sum-of-multiples/description.md]: This is almost identically repeated at the end of the description. "particular numbers" also seems to be confusing language here.

File: exercises/bank-account/description.md [hunk 8]
```diff
@@ -1,4 +1,4 @@
-Simulate a bank account supporting opening/closing, withdraws, and deposits of money. Watch out for concurrent transactions!
+Simulate a bank account supporting opening/closing, withdrawals, and deposits of money. Watch out for concurrent transactions!
```
Comment (stkent) [inline exercises/bank-account/description.md]: Needs formatting for width.

File: exercises/flatten-array/description.md [hunk 9]
```diff
@@ -1,4 +1,4 @@
-Take a nested list and return a single list with all values except nil/null.
+Take a nested list and return a single flattened list with all values except nil/null.
```
Comment (stkent) [inline exercises/flatten-array/description.md]: Needs formatting for width?

File: exercises/secret-handshake/description.md [hunk 10]
```diff
@@ -16,6 +14,8 @@ binary decide to come up with a secret "handshake".
 10000 = Reverse the order of the operations in the secret handshake.
 ```
 
+Given a decimal number, convert it to the appropriate sequence of events for a secret handshake.
```
Comment (stkent) [inline exercises/secret-handshake/description.md]: Needs formatting for width.

File: exercises/transpose/description.md [hunk 11]
```diff
@@ -56,4 +54,4 @@ BE
 
 In general, all characters from the input should also be present in the transposed output.
 That means that if a column in the input text contains only spaces on its bottom-most row(s), 
-the corresponding output row should contain the spaces in its right-most column(s).
\ No newline at end of file
+the corresponding output row should contain the spaces in its right-most column(s).
```
Comment (stkent) [inline exercises/transpose/description.md]: Needs formatting for width.

File: exercises/two-bucket/description.md [hunk 12]
```diff
@@ -27,4 +25,4 @@ To conclude, the only valid moves are:
 - emptying one bucket and doing nothing to the other
 - filling one bucket and doing nothing to the other
 
-Written with <3 at [Fullstack Academy](http://www.fullstackacademy.com/) by [Lindsay](http://lindsaylevine.com).
\ No newline at end of file
+Written with <3 at [Fullstack Academy](http://www.fullstackacademy.com/) by [Lindsay](http://lindsaylevine.com).
```
Comment (stkent) [inline exercises/two-bucket/description.md]: Needs formatting for width.

File: exercises/protein-translation/description.md [hunk 13]
```diff
@@ -38,3 +36,6 @@ UAU, UAC              | Tyrosine
 UGU, UGC              | Cysteine
 UGG                   | Tryptophan
 UAA, UAG, UGA         | STOP
+
+
+Learn more about [protein translation on Wikipedia](http://en.wikipedia.org/wiki/Translation_(biology))
```
Comment (stkent) [inline exercises/protein-translation/description.md]: Needs formatting for width.

File: exercises/book-store/description.md (hunk 5 above)
Comment (stkent) [inline exercises/book-store/description.md]: +1

**Comment by stkent** [comment]
@Insti reviewed as requested!

File: exercises/pangram/description.md [hunk 14]
```diff
@@ -1,6 +1,7 @@
 Determine if a sentence is a pangram. A pangram (Greek: παν γράμμα, pan gramma,
 "every letter") is a sentence using every letter of the alphabet at least once.
-The best known English pangram is "The quick brown fox jumps over the lazy dog."
+The best known English pangram is: 
+"The quick brown fox jumps over the lazy dog."
```
Comment (stkent) [inline exercises/pangram/description.md]: Could use markdown quote syntax here: `>`?

File: exercises/sum-of-multiples/description.md (hunk 7 above)
Comment (Insti) [inline exercises/sum-of-multiples/description.md]: This is a good point but I'm not going to change this as part of this PR as it is not related to the initial duplication.

File: exercises/flatten-array/description.md (hunk 9 above)
Comment (Insti) [inline exercises/flatten-array/description.md]: Yes, but not initial duplication related.

File: exercises/secret-handshake/description.md (hunk 10 above)
Comment (Insti) [inline exercises/secret-handshake/description.md]: Yes, but not initial duplication related.

File: exercises/protein-translation/description.md (hunk 13 above)
Comment (Insti) [inline exercises/protein-translation/description.md]: The rendered view has no width issues?

File: exercises/pangram/description.md (hunk 14 above)
Comment (Insti) [inline exercises/pangram/description.md]: Good idea.

**Comment by Insti** [comment]
Thanks @stkent 

Does somebody want to approve these changes so I can squash/merge them?

**Comment by ErikSchierboom** [comment]
@Insti looks good to me!

**Comment by Insti** [comment]
Thanks @ErikSchierboom, @rpottsoh & @stkent 

I think this completes the separation of blurbs and descriptions.

Any further changes to required to the content of the descriptions can be done as needed.

**Pull request merged**